src/chains/ethereum/transaction/src/eip1559-fee-market-transaction.ts

```diff
@@ -253,8 +253,8 @@ export class EIP1559FeeMarketTransaction extends RuntimeTransaction {
     return computeIntrinsicsFeeMarketTx(v, <EIP1559FeeMarketDatabaseTx>raw);
   }
 
-  public updateEffectiveGasPrice(baseFeePerGas: Quantity) {
-    const baseFeePerGasBigInt = baseFeePerGas.toBigInt();
+  public updateEffectiveGasPrice(baseFeePerGas: bigint) {
+    const baseFeePerGasBigInt = baseFeePerGas;
```

Comment:
Instead of reassigning this variable can we just use the `baseFeePerGas` variable as is?

Reply:
Yes, obviously. :sweat_smile: